spec/concurrent/struct_shared.rb

```diff
@@ -26,6 +26,18 @@
       expect(clazz.ancestors).to include described_class
     end
 
+    it 'ignores methods on ancestor classes' do
+      ancestor = described_class.ancestors.first
+      ancestor.class_eval { def foo; end }
+
+      clazz = described_class.new(:foo)
+      expect{ described_class.const_get(clazz.to_s) }.to raise_error(NameError)
+      expect(clazz).to be_a Class
+      expect(clazz.ancestors).to include described_class
```

Comment:
This seems not meaningful to test here, let's just that that creating an instance of it plus accessing that member works as expected.